python/python/glide/async_commands/core.py

```diff
@@ -2326,6 +2326,33 @@ async def zscore(self, key: str, member: str) -> Optional[float]:
             await self._execute_command(RequestType.ZScore, [key, member]),
         )
 
+    async def zmscore(
+        self,
+        key: str,
+        members: List[str],
+    ) -> List[float]:
```

Comment:
```suggestion
    ) -> List[Optional[float]]:
```

Reply:
The redis docs say the command has a "null reply: if the member does not exist in the sorted set", however it does not seem like a null value is ever returned when I use redis-cli. If I specify a non-existing key I get `1) (nil)` as a response instead of just `(nil)`, and if I specify an existing key with one non-existing member I also get `1) (nil)`. I tried with both RESP2 and RESP2 and had the same results.

Reply:
Oh good point, thanks for pointing that out!